### PR TITLE
Install/upgrade instructions for 7.1

### PR DESCRIPTION
**Pull request by begriffs**
Needs testing when new packages are released, as part of #528

### CODE REVIEW

File: installation/single_machine_rhel.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Let's create directories for those nodes to store their data. For convenience in
   sudo su - postgres
 
   # include path to postgres binaries
-  export PATH=$PATH:/usr/pgsql-9.6/bin
+  export PATH=$PATH:/usr/pgsql-10.0/bin
```
Comment (sumedhpathak) [inline installation/single_machine_rhel.rst]: This should be ```/usr/pgsql-10/bin```

Comment (begriffs) [reply installation/single_machine_rhel.rst]: K. That's how it was on the debian side too.

File: installation/production_rhel.rst [hunk 2]
```diff
@@ -26,21 +26,21 @@ Steps to be executed on all nodes
 ::
 
   # install PostgreSQL with Citus extension
-  sudo yum install -y citus70_96
+  sudo yum install -y citus71_10
   # initialize system database (using RHEL 6 vs 7 method as necessary)
-  sudo service postgresql-9.6 initdb || sudo /usr/pgsql-9.6/bin/postgresql96-setup initdb
+  sudo service postgresql-10.0 initdb || sudo /usr/pgsql-10.0/bin/postgresql96-setup initdb
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: At least for ```sudo /usr/pgsql-10.0/bin/postgresql96-setup initdb```, that should be 
```sudo /usr/pgsql-10/bin/postgresql-10-setup initdb```.

File: installation/production_rhel.rst [hunk 3]
```diff
@@ -26,21 +26,21 @@ Steps to be executed on all nodes
...
   # preload citus extension
-  echo "shared_preload_libraries = 'citus'" | sudo tee -a /var/lib/pgsql/9.6/data/postgresql.conf
+  echo "shared_preload_libraries = 'citus'" | sudo tee -a /var/lib/pgsql/10.0/data/postgresql.conf
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: Should be 
```echo "shared_preload_libraries = 'citus'" | sudo tee -a /var/lib/pgsql/10/data/postgresql.conf```
(10.0 -> 10)

File: installation/production_rhel.rst [hunk 4]
```diff
@@ -26,21 +26,21 @@ Steps to be executed on all nodes
...
 
-PostgreSQL adds version-specific binaries in `/usr/pgsql-9.6/bin`, but you'll usually just need psql, whose latest version is added to your path, and managing the server itself can be done with the *service* command.
+PostgreSQL adds version-specific binaries in `/usr/pgsql-10.0/bin`, but you'll usually just need psql, whose latest version is added to your path, and managing the server itself can be done with the *service* command.
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: Should be `/usr/pgsql-10/bin`

File: installation/production_rhel.rst [hunk 5]
```diff
@@ -26,21 +26,21 @@ Steps to be executed on all nodes
...
 
 **3. Configure connection and authentication**
 
 Before starting the database let's change its access permissions. By default the database server listens only to clients on localhost. As a part of this step, we instruct it to listen on all IP interfaces, and then configure the client authentication file to allow all incoming connections from the local network.
 
 ::
 
-  sudo vi /var/lib/pgsql/9.6/data/postgresql.conf
+  sudo vi /var/lib/pgsql/10.0/data/postgresql.conf
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: 10.0 -> 10

File: installation/production_rhel.rst [hunk 6]
```diff
@@ -49,7 +49,7 @@ Before starting the database let's change its access permissions. By default the
 
 ::
 
-  sudo vi /var/lib/pgsql/9.6/data/pg_hba.conf
+  sudo vi /var/lib/pgsql/10.0/data/pg_hba.conf
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: 10.0 -> 10

File: installation/production_rhel.rst [hunk 7]
```diff
@@ -69,9 +69,9 @@ Before starting the database let's change its access permissions. By default the
 ::
 
   # start the db server
-  sudo service postgresql-9.6 restart
+  sudo service postgresql-10.0 restart
```
Comment (sumedhpathak) [inline installation/production_rhel.rst]: 10.0 -> 10

**Pull request merged**